wallet/commands.go

```diff
@@ -173,6 +173,44 @@ func (wal *Wallet) GetAllTransactions(offset, limit, txfilter int32) {
 	}()
 }
 
+// GetTransactionsByWallet get list of transactions fitting the parameters.
+// It is non-blocking and sends its result or any error to wal.Send.
+func (wal *Wallet) GetTransactionsByWallet(walletID int, offset, limit, txfilter int32) {
+	go func() {
+		var resp Response
+
+		wallets, err := wal.wallets()
+
+		if err != nil {
+			resp.Err = err
+			wal.Send <- resp
+
+			return
+		}
+
+		var alltxs []dcrlibwallet.Transaction
+
+		for _, wall := range wallets {
+			if wall.ID == walletID {
```

Comment:
You can query the multiwallet directly for a wallet with `wal.multi.WalletWithID`